lib/krane/deploy_task.rb

```diff
@@ -191,7 +193,7 @@ def resource_deployer
     end
 
     def kubeclient_builder
-      @kubeclient_builder ||= KubeclientBuilder.new
+      @kubeclient_builder ||= KubeclientBuilder.new(kubeconfig: @task_config.kubeconfig)
```

Comment:
This is now in a lot of places, so don't love that. Seems like every time we're creating a kubeclientbuilder object from a task config, so why don't we make that a method on the task config that will return a kubeclientbuilder. That way we don't have to pass a task_config around too often.